### PR TITLE
Eng 1599 workflow status bar popover

### PR DESCRIPTION
**Pull request by agiron123**
## Describe your changes and why you are making these changes
This PR takes the workflow status bar that was on the right side of the workflow details screen and turns it into a popover which is now located in the workflow header.

## Related issue number (if any)
ENG-1599

## Loom Demo:
https://www.loom.com/share/b0407dcc1195480f834204b41705c35a

## Checklist before requesting a review
- [x] I have created a descriptive PR title. The PR title should complete the sentence "This PR...".
- [x] I have performed a self-review of my code.
- [x] I have included a small demo of the changes. For the UI, this would be a screenshot or a Loom video.
- [x] If this is a new feature, I have added unit tests and integration tests.
- [x] I have run the integration tests locally and they are passing.
- [x] I have run the linter script locally (See `python3 scripts/run_linters.py -h` for usage).
- [x] All features on the UI continue to work correctly.
- [x] Added one of the following CI labels:
    - `run_integration_test`: Runs integration tests
    - `skip_integration_test`: Skips integration tests (Should be used when changes are ONLY documentation/UI)

### CODE REVIEW

File: src/ui/common/src/components/pages/workflow/id/index.tsx [hunk 1]
```diff
@@ -368,7 +367,7 @@ const WorkflowPage: React.FC<WorkflowPageProps> = ({
         </Drawer>
       )}
 
-      <WorkflowStatusBar user={user} />
+      {/* <WorkflowStatusBar user={user} /> */}
```
Comment (agiron123) [inline src/ui/common/src/components/pages/workflow/id/index.tsx:370]: remove this line.

File: src/ui/common/src/components/workflows/StatusBar.tsx [hunk 2]
```diff
@@ -54,17 +54,21 @@ type WorkflowStatusItem = {
...
 
 interface ActiveWorkflowStatusTabProps {
+  setActiveWorkflowStatusTab: (tab: WorkflowStatusTabs) => void;
   activeWorkflowStatusTab: string;
   listItems: WorkflowStatusItem[];
 }
 
-export const StatusBarHeaderHeightInPx = 50;
+export const StatusBarHeaderHeightInPx = 41;
+//export const StatusBarHeaderHeightInPx = 82;
```
Comment (agiron123) [inline src/ui/common/src/components/workflows/StatusBar.tsx:63]: remove comments in this file :)

**Comment by vsreekanti** [comment]
@agiron123, this looks awesome. One small design nit -- can we reduce the space between/around the items in the status bar menu? It looks a little wide/overly spaced out to me.

**Comment by agiron123** [comment]
> @agiron123, this looks awesome. One small design nit -- can we reduce the space between/around the items in the status bar menu? It looks a little wide/overly spaced out to me.

@vsreekanti sure, I can reduce the spacing. will post a screenshot.

**Comment by agiron123** [comment]
@vsreekanti Latest screenshot can be found here:
image

**Comment by vsreekanti** [comment]
Thanks @agiron123! Any opinions on reducing the spaces in between the items / to the down chevron? I'm curious what you think.

I think I also mentioned this on Friday, but in case I forgot, can we make it so that when there's a "No errors/warnings/etc.", the rest popover is only one item tall? It looks a little funny IMO to have a long popover with one small entry. 

Great work!

**Comment by agiron123** [comment]
> Thanks @agiron123! Any opinions on reducing the spaces in between the items / to the down chevron? I'm curious what you think.
> 
> I think I also mentioned this on Friday, but in case I forgot, can we make it so that when there's a "No errors/warnings/etc.", the rest popover is only one item tall? It looks a little funny IMO to have a long popover with one small entry.
> 
> Great work!

@vsreekanti sure thing, will get this resolved before merging.

**Comment by agiron123** [comment]
Latest Screenshot:

image

image

**Pull request merged**